uxarray/helpers.py

```diff
@@ -673,3 +673,53 @@ def close_face_nodes(Mesh2_face_nodes, nMesh2_face, nMaxMesh2_face_nodes):
     np.put(closed.ravel(), first_fv_idx_1d, first_node_value)
 
     return closed
+
+
+def _convert_face_node_conn_to_sparse_matrix(
+        dense_matrix: np.ndarray, fill_value=INT_FILL_VALUE) -> tuple:
```

Comment:
Rename to `dense_face_nodes` or similar to indicate this function is only used for face nodes

Reply:
the current function name already has the `face_nodes` keywords inside. I think it already refect the purpose